src/c++/perf_analyzer/schema.json

```diff
@@ -0,0 +1,96 @@
+{
+    "$schema": "https://json-schema.org/draft/2020-12/schema",
+    "$id": "https://github.com/triton-inference-server/client/blob/main/src/c%2B%2B/perf_analyzer/examples/schema.json",
+    "title": "Perf Analyzer output data",
+    "description": "A json file describing the output from a perf analyzer run.",
+    "type": "object",
+    "required": [
+        "experiments",
+        "version"
+    ],
+    "properties": {
+        "experiments": {
+            "description": "The array of all experiments run by perf analyzer.",
+            "type": "array",
+            "required": [
+                "experiment",
+                "requests",
+                "window_boundaries"
+            ],
+            "minItems": 1,
+            "uniqueItems": true,
+            "items": {
+                "type": "object",
+                "properties": {
+                    "experiment": {
+                        "description": "A single experiment run by perf analyzer.",
+                        "type": "object",
+                        "required": [
+                            "mode",
+                            "value"
+                        ],
+                        "minItems": 1,
+                        "maxItems": 1,
+                        "properties": {
+                            "mode": {
+                                "description": "Operating mode of perf analyzer: For example, concurrency or request rate.",
+                                "type": "string"
+                            },
+                            "value": {
+                                "description": "Concurrency or request rate for the current experiment.",
+                                "type": "number"
+                            }
+                        }
+                    },
+                    "requests": {
+                        "description": "The array of requests sent by perf_analyzer for this experiment.",
+                        "type": "array",
+                        "uniqueItems": true,
```

Comment:
I'm not sure if it's necessary to declare the elements as unique. Idk how specific the timestamps are, but it might be possible that two requests start at exactly the same time and have responses coming in at exactly the same time.

Reply:
I can remove this constraint. I thought about that case but thought it was infinitesimally small. If you think its a concern, we can remove it.

Reply:
I agree it's extremely unlikely that two requests would have the same start time and response times, but since this schema is supposed to represent the real-world output file, I would say we should remove this unique constraint, as there is no such constraint in PA when it records timestamps and outputs this json.

Reply:
Removed